source/module_esolver/esolver_ks.cpp

```diff
@@ -382,49 +380,75 @@ void ESolver_KS<T, Device>::hamilt2density(const int istep, const int iter, cons
 //! mohan add 2024-05-11
 //------------------------------------------------------------------------------
 template <typename T, typename Device>
-void ESolver_KS<T, Device>::print_wfcfft(Input& inp, std::ofstream& ofs)
-{
+void ESolver_KS<T, Device>::print_wfcfft(Input& inp, std::ofstream& ofs) {
     ofs << "\n\n\n\n";
```

Comment:
@haozhihan could we remove this rule from clang-format?

Reply:
I think so, should do this asap.